lib/ts/recipe/thirdpartyemailpassword/types.ts

```diff
@@ -90,7 +90,8 @@ export type RecipeInterface = {
             fromUserInfoAPI?: { [key: string]: any };
         };
         tenantId: string;
-        userContext: any;
+        shouldAttemptAccountLinkingIfAllowed: boolean;
```

Comment:
this should not be there.

Reply:
check the entire code back for this boolean and remove it propely. It should only be there in manuallyCreateOrUpdateUser recipe function (except for the index function arg).

Reply:
Done